flexget/api/core/server.py

```diff
@@ -130,19 +133,15 @@ class ServerReloadAPI(APIResource):
     @api.response(501, model=yaml_error_schema, description='YAML syntax error')
     @api.response(502, model=config_validation_schema, description='Config validation error')
     @api.response(200, model=base_message_schema)
-    def post(self, session=None):
+    def post(self, session: None = None) -> Response:
         """ Manage server operations """
         data = request.json
         if data['operation'] == 'reload':
             try:
                 self.manager.load_config(output_to_console=False)
             except YAMLError as e:
-                if (
-                    hasattr(e, 'problem')
-                    and hasattr(e, 'context_mark')
-                    and hasattr(e, 'problem_mark')
-                ):
-                    error = {}
+                if isinstance(e, MarkedYAMLError):
+                    error: Dict[str, int] = {}
```

Comment:
seems kind of overkill to me to specify such an internal variable type, but ok

Reply:
Dict (and other collections) annotations are sometimes required, when mypy cannot infer the type of the dict based on its usage. This is such a hint, to stop it from complaining,